### DISCUSSION TITLE
validation of wrf model output(LH and SHF)

### DISCUSSION THREAD

**Discussion by neha12-3** [NCAR/hrldas#178] [Q&A] [answered]
i want to validate my wrf model output (LH nad SHF).Can u please suggest open access data with which i 
can validate my output

**Comment by cenlinhe** [comment, marked as answer]
It depends on which study region you are looking at. There may be some ground based flux stations in your specific domain that you could use. You could also use the FluxNet global network (https://fluxnet.org/data/) to select the flux tower sites in your domain. You could also use the model-based reanalysis dataset GLEAM (https://www.gleam.eu/).

**Reply by neha12-3** [reply]
ok thnku

**Comment by neha12-3** [comment]
![1714971330332](https://github.com/NCAR/hrldas/assets/167417195/a39a88a2-5e3b-4152-b2f1-c2950e991a60)
i have got this error while running wrf.exe.why this?

**Reply by cenlinhe** [reply]
Please post WRF-related questions on the WRF forum: https://forum.mmm.ucar.edu/
thanks!

**Comment by neha12-3** [comment]
can you help me how to perform ensemble experiment with different physical  parametriztion scheme

**Reply by cenlinhe** [reply]
Hi, do you mean WRF experiment or HRLDAS/NoahMP experiment? If it is WRF, we suggest you post the questions on the WRF forum: https://forum.mmm.ucar.edu/
If it is NoahMP, you can change the different physics options in the Noah-MP namelist to run different ensemble experiment.

**Comment by neha12-3** [comment]
error while running wrf.exe with assimilated wrfinput for domain3 
![Uploading 1749802919302.jpg…]()
